src/components/Map/Map.tsx

```diff
@@ -92,22 +92,39 @@ export default function Map(props: MapProps): JSX.Element {
     [onTokenExpired]
   );
 
+  const clearFeatureVar = (featureVar: any, property?: string) => {
+    const map: any = mapRef && mapRef.current;
+
+    Object.keys(featureVar).forEach((sourceId) => {
+      Object.keys(featureVar[sourceId]).forEach((id) => {
+        if (map && property) {
+          map.setFeatureState({ source: sourceId, id: featureVar[sourceId][id] }, { [property]: false });
+        }
+        delete featureVar[sourceId][id];
```

Comment:
not a biggie,, the delete on 105 will take care of this

Reply:
Oh, i suppose the GC will get it after the referencing variable has been deleted. Okay